app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java

```diff
@@ -141,11 +144,58 @@ public void exportCSV() {
                 writer.writeNext(line);
             }
             writer.close();
+            addNotification(this,f);
             Toast.makeText(this, R.string.txt_history_exported, Toast.LENGTH_LONG).show();
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
+    private void addNotification(Context context,File file){
+
+        NotificationCompat.Builder builder =
+                new NotificationCompat.Builder(this,"notification_channel")
+                        .addAction(R.drawable.ic_share_black_24dp, getString(R.string.share_notification), shareFileIntent(context,file))
+                        .setSmallIcon(R.mipmap.ic_launcher)
+                        .setContentTitle(getString(R.string.download_complete))
+                        .setContentText(file.toString());
+
+        Intent notificationIntent = openFile(file);
+        PendingIntent contentIntent = PendingIntent.getActivity(this, 0, notificationIntent,
+                PendingIntent.FLAG_UPDATE_CURRENT);
+        builder.setContentIntent(contentIntent);
+
+        // Add as notification
+        NotificationManager manager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        manager.notify(0, builder.build());
```

Comment:
Got a linting warning here, because `manager` can be `null`, can you surround it with `if (manager != null)`?

Reply:
thanks for the remarks, yes we should,  i will do it.